### PR TITLE
fix: unshiedling

### PR DESCRIPTION
**Pull request by mateuszjasiuk**
Bump namada
Fix unshielding
Fix e2e tests(need to double check)

### CODE REVIEW

**Comment by phy-chain** [comment]
@mateuszjasiuk :wave: 
I see that you're adding shielded-sync to the extension, nice !
Is that the reason why currently .insert_spending_key does not work ?
I've tried inserting an xsk in my extension wallet but even if the function call seems to work, nothing actually happens. I dont see the key in my accounts

**Comment by mateuszjasiuk** [comment]
> @mateuszjasiuk 👋 I see that you're adding shielded-sync to the extension, nice ! Is that the reason why currently .insert_spending_key does not work ? I've tried inserting an xsk in my extension wallet but even if the function call seems to work, nothing actually happens. I dont see the key in my accounts

I don't think this is related. Inserting spending key "should" work. We were not using that function really for some time. It's hard to judge your problem without taking a look at what you are trying to do :)

**Comment by phy-chain** [comment]
> > @mateuszjasiuk 👋 I see that you're adding shielded-sync to the extension, nice ! Is that the reason why currently .insert_spending_key does not work ? I've tried inserting an xsk in my extension wallet but even if the function call seems to work, nothing actually happens. I dont see the key in my accounts
> 
> I don't think this is related. Inserting spending key "should" work. We were not using that function really for some time. It's hard to judge your problem without taking a look at what you are trying to do :)

So far, I've added a small "add spending key" flow in the extension, called the `sdk.add_spending_key` function, and after some effort, I got it to "work", meaning that the calls go through without failing/raising an error in the service worker, but then the address does not show anywhere (in my account for instance). Am I missing something ? Should I link it to my account or something ?

File: apps/namada-interface/src/slices/accounts.ts [hunk 1]
```diff
@@ -193,18 +193,55 @@ const balancesAtom = (() => {
...
-        const result = await namada.queryBalances(account.address, [
-          nativeToken || tokenAddress,
-        ]);
-        const balance = result.reduce(
-          (acc, curr) => ({ ...acc, [curr.token]: new BigNumber(curr.amount) }),
-          {} as Balance
-        );
-        set(base, { ...get(base), [account.address]: balance });
+      // Split accounts into transparent and shielded
+      const [transparentAccounts, shieldedAccounts] = accounts.reduce((acc, curr) => {
```
Comment (phy-chain) [inline apps/namada-interface/src/slices/accounts.ts:197]: How do you create an account from inserting a spending key ? 
It  seems that this is my main issue with adding a spending key, it is not registered in the "accounts'

Comment (phy-chain) [reply apps/namada-interface/src/slices/accounts.ts:197]: Sure, i''ve pinged you on discord MP !

**Comment by phy-chain** [comment]
@mateuszjasiuk I have tried to run your branch, and my service worker is spamming thousands of requests to my RPC. When I load the page, it starts loading shielded context at block 1, fires queries like mad, reaches height around 15000 or so, then the service worker blows and I have to refresh, goes to inactive, and I never get to the end of the sync. Do you encounter this problem ?

**Review by emccorson — APPROVED** [review]
Looks good to me! Just left some minor comments.

**Comment by mateuszjasiuk** [comment]
> @mateuszjasiuk I have tried to run your branch, and my service worker is spamming thousands of requests to my RPC. When I load the page, it starts loading shielded context at block 1, fires queries like mad, reaches height around 15000 or so, then the service worker blows and I have to refresh, goes to inactive, and I never get to the end of the sync. Do you encounter this problem ?

There is no better way for now, we have to query each block. We usually are not testing against long running testnets but rather local chains. This PR works in principle. It will be improved once there are further improvements to the namada sdk

**Pull request merged**